packages/form-js-editor/src/features/properties-panel/entries/NumberEntries.js

```diff
@@ -91,7 +91,13 @@ function NumberArrowStep(props) {
     return value;
   };
 
-  const setValue = (value) => editField(field, [ 'increment' ], value);
+  const clearLeadingZeroes = (value) => {
+    if (!value) return value;
+    const trimmed = value.replace(/^0+/g, '');
+    return (trimmed.startsWith('.') ? '0' : '') + trimmed;
```

Comment:
```suggestion
    return `${trimmed.startsWith('.') ? '0' : ''}${trimmed}`;
```

Reply:
I think this is personal preference, but personally, I wouldn't use computed strings for concatenation. So I'll merge as-is. Thanks for the review :)